docs/reference/components/sources/http.cue

```diff
@@ -107,8 +107,38 @@ components: sources: http: {
 				}
 			}
 		}
+		path: {
+			common:      false
+			description: "The URL path on which log event POST request shall be sent."
+			required:    false
+			type: string: {
+				default: "/"
+				examples: ["/event/path", "/logs"]
+				syntax: "literal"
+			}
+		}
+		strict_path: {
+			common: false
+			description: """
+				If set to true only request using the exact URL path specified in path will be accepted,
+				else event log sent to a URL path that starts with the value of path will be accepted.
+				With strict_path set to false and path set to \"/\" the configured HTTP source will
+				accept log events from any URL path.
```

Comment:
We have also preferred to back-quote literals and config fields:
```suggestion
				If set to `true`, only requests using the exact URL path specified in `path` will be accepted;
				otherwise requests sent to a URL path that starts with the value of `path` will be accepted.
				With `strict_path` set to `false` and `path` set to `""`, the configured HTTP source will
				accept requests from any URL path.
```